.lighthouserc.json

```diff
@@ -0,0 +1,39 @@
+{
+  "ci": {
+    "collect": {
+      "startServerCommand": "npm run serve",
+      "startServerReadyPattern": "Server running at http://localhost:3000/",
+      "url": [
+        "http://localhost:3000/en",
+        "http://localhost:3000/ar",
+        "http://localhost:3000/ca",
+        "http://localhost:3000/de",
+        "http://localhost:3000/es",
+        "http://localhost:3000/fa",
+        "http://localhost:3000/fr",
+        "http://localhost:3000/id",
+        "http://localhost:3000/it",
+        "http://localhost:3000/ja",
+        "http://localhost:3000/ka",
+        "http://localhost:3000/ko",
+        "http://localhost:3000/pt-br",
+        "http://localhost:3000/ro",
+        "http://localhost:3000/ru",
+        "http://localhost:3000/tr",
+        "http://localhost:3000/uk",
+        "http://localhost:3000/zh-cn",
+        "http://localhost:3000/zh-tw"
```

Comment:
I suspect we only need to test `/en` - the rest will exhibit similar experiences but with different content.

Reply:
Changed to more useful URLs